Changelog.md

```diff
@@ -10,6 +10,7 @@
 - Updated `mapJSON` API to include an optional named parameter `failsOnEmptyData:` that when overriden returns an empty `NSNull()` result instead of throwing an error when the response data is empty.
 - Added `supportsMultipart` to the `Method` type, which helps determine whether to use `multipart/form-data` encoding.
 - Added `PATCH` and `CONNECT` to the `Method` cases which support multipart encoding.
+- Added `request` for `Rsponse`.
```

Comment:
Maybe also add an explanation for the addition; right now, it doesn't really say anything.